assets/js/vendor/cards-expandable.js

```diff
@@ -0,0 +1,12 @@
+// JS for expandable cards (.va-card--expands)
+$(document).ready(function() {
+	$('.va-card-body').on('click', function(e){
+    	if(e.target.nodeName == 'DIV') {
```

Comment:
Using `is('div')` makes sense. The problem with using `this` is that the click target might be a child of `.va-card-body`. But checking the `currentTarget` might be better here.